tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -121,6 +130,37 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Set cluster_kubeconfig value for 'drpc_obj' object to fetch the details without switching the cluster context
+        acm_cluster_kubeconfig = os.path.join(
+            config.clusters[get_active_acm_index()].ENV_DATA["cluster_path"],
+            config.clusters[get_active_acm_index()].RUN.get("kubeconfig_location"),
+        )
+        drpc_obj.cluster_kubeconfig = acm_cluster_kubeconfig
+
+        # Get lastGroupSyncTime before failover
+        drpc_data = drpc_obj.get()
+        last_group_sync_time = drpc_data.get("status").get("lastGroupSyncTime")
+        logger.info(
+            f"The value of lastGroupSyncTime before failover is {last_group_sync_time}."
```

Comment:
We aren't fetching this value again for Relocate, so the logs could be changed to
```suggestion
            f"The value of lastGroupSyncTime before failover or relocate is {last_group_sync_time}."
```

Reply:
The value of lastGroupSyncTime will be fetched after failover(after bringing up the primary cluster) and after relocate.